src/lightdark2dlpdm.jl

```diff
@@ -124,7 +124,7 @@ POMDPs.actions(p::LightDark2DLpdm, ::LD2State) = actions(p::LightDark2DLpdm)
 
 LPDM.max_actions(pomdp::LightDark2DLpdm) = pomdp.max_actions
 
-# For "simulated annealing" which is adaptive.
+# For "simulated annealing" which is adaptive. 
```

Comment:
So in order to make the continuous action space stuff work for refinement and synthesis, do I need to re-write this function or can I leave it as is, and just add the necessary properties that it references that we don't currently define in refinement(n_new_actions, standard action space, action_limits, action_range_fraction). If this function is not general enough to allow for me to just make some simple changes, we may need to sit down and talk about it because I am not too familiar with that we are actually trying to do in this function. @ebalaban if it's easier to talk via phone or in person, let me know.